api/main_endpoints/routes/User.js

```diff
@@ -235,6 +238,14 @@ router.post('/edit', async (req, res) => {
         .status(NOT_FOUND)
         .send({ message: `${query.email} not found.` });
     }
+
+    AuditLog.create({
+      userId: decoded._id, // the user making the update
+      action: AuditLogActions.UPDATE_USER,
+      documentId: user._id, // the user affected by the update
+      details: {updatedInfo: user}
```

Comment:
- edit a user, change their password
- does the password field appear in the AuditLog collection?
- if so, lets add logic to replace password to `true`, if it is set, so we avoid storing it

Reply:
add a unit test to verify this as well